.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-watsonx-ai-saas
+    kind: solution
+    catalog_id: b5c105a0-04ac-4418-ab2a-3696f7f1d96d
+    offering_id: 2cc49ba4-5612-4c86-b171-d9b4b34fc52d
```

Comment:
As per slack discussions, this offering ID need to point to a new offering in your catalog, since old offering is hooked up with internal GHE token

Reply:
done